teflo_webhooks_notification_plugin/templates/slack_template.jinja

```diff
@@ -38,7 +38,23 @@
                 ]
             },
         {% endif %}
-
+        {% if scenario.get_all_assets() %}
```

Comment:
It does not have an else part. if scenario.get_all_assets() is not present

Reply:
done